# System version information in hash (binary) format

uint64 timestamp		# time since system start (microseconds)
uint64 hw_version		# main autopilot hw version
uint64 hw_revision		# main autopilot hw revision
uint64 soc_arch_id		# main autopilot soc architecture identifier
uint64 sw_version		# main autopilot sw version
uint64 os_version		# OS or middleware version
uint64 bl_version		# bootloader version
uint64 component_version1	# vendor specific component version
uint64 component_version2	# vendor specific component version
